src/cpp/session/modules/rmarkdown/NotebookAlternateEngines.cpp

```diff
@@ -338,6 +338,49 @@ Error executeStanEngineChunk(const std::string& docId,
    return Success();
 }
 
+Error executeSqlEngineChunk(const std::string& docId,
```

Comment:
@jmcphers: I'm relying on `events().onDataOutput(FilePath(targetPath), FilePath());` to signal the output of the SQL chunk. However, this initially didn't work since for alternate engines, we don't connect all the events as we do for the R engine and therefore, `onDataOuput` was not being triggered. To solve this, now I'm connecting the execution context in `NotebookQueue.cpp`. So far so good; however, this is causing subsequent chunks to hang after executing a SQL chunk. Haven't debugged this further yet so it would be great if you can share your thoughts before I follow this route further. Thanks!